starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -841,11 +858,12 @@ sub large_dither_checks {
             last;
         }
     }
-    # Is dither disabled at EOM and one minute before?
-    if ((abs($obs_tstart - $obs_start_dither->{time} - 60) > $time_tol)
-            or ($obs_start_dither->{state} ne 'DISA')){
+
+    my $det = (($self->{SI} eq 'HRC-S') or ($self->{SI} eq 'HRC-I')) ? 'hrc' : 'acis';
+    # Is dither nominal for detector at EOM
+    if ($det ne standard_dither($obs_start_dither)){
         push @{$self->{warn}},
-            sprintf("$alarm Dither should be disabled 1 min before obs start for Large Dither\n");
+            sprintf("$alarm Dither should be detector nominal 1 min before obs start for Large Dither\n");
```

Comment:
Does it need to be `detector nominal` or just not large?  The large dither rules are about PCAD and ACA, so I don't think the detector matters.

Reply:
I think it needs to be *a* nominal pattern (not large but with the wrong period would be another issue).  AFAIK anything other than detector nominal dither would speak to either the dither states getting screwed up or DOT manual edits when they should not be required in ORViewer.  Might be safest to just confirm it is detector-nominal dither.

Reply:
Another topic for SSAWG.  I don't see why 4x4 dither leading up to a large dither observation would be a problem.  The perennial question, are we checking that the loads are safe for the spacecraft or are we checking the planning process?

Reply:
In this case, I still think it is easier to check the planning process.  But I can add a check on rates.

Reply:
You don't need to check rates, that is PCAD not ACA.  Just amplitude, which is what matters for ACA.

Reply:
For ACA we don't care at all if the proper dither is used for the proseco.acq .

But starcheck output is also used as a PCAD extra check in other areas as well.